scripts/run_docker_gpu.sh

```diff
@@ -6,7 +6,14 @@ cmd_line="$@"
 echo "Executing in the docker (gpu image):"
 echo $cmd_line
 
+# TODO: always use new-style once sufficiently widely used (probably 2021 onwards)
+if [ -x "$(which nvidia-docker)" ]; then
```

Comment:
so here, you are just checking if `nvidia-docker` exists?

Reply:
Yes, or more accurately "if `nvidia-docker` is executable"

Reply:
so if I get it right, in the newest docker version, there is no need for an additional executable?